lib/livebook/web_socket/client.ex

```diff
@@ -180,11 +180,17 @@ defmodule Livebook.WebSocket.Client do
 
   defp handle_frames(response, frames) do
     Enum.reduce(frames, response, fn
+      _, {_, _} = acc ->
+        acc
+
       {:binary, binary}, acc ->
         {:ok, %{acc | body: binary}}
 
       {:close, _code, _data}, acc ->
         {:close, acc}
+
+      _, acc ->
+        {:ok, acc}
     end)
   end
```

Comment:
```suggestion
  defp handle_frames(response, frames) do
    %{response | body: IO.iodata_to_binary(handle_frames_body(frames))}
  end

  defp handle_frames_body([{:binary, binary} | rest]), do: [binary | handle_frames_body(rest)]
  defp handle_frames_body([{:close, _, _}]), do: []
  defp handle_frames_body([]), do: []
```

Reply:
This wouldn't work since we receive the entire message, so we should only discard the `rest`. Also, we can't use `IO.iodata_to_binary/1` if we're going to decode with `<ProtobufMessageStruct>.decode/1`

Reply:
Is it a single message maybe because the message is too small? but it can be multiple frames if large? We would need to analyze the mint.websocket code or docs to make sure if one server frame is equal one client frame.

Also i am not sure why `iodata_to_binary/1` does not work since it returns a binary as in the current code?

Reply:
If we want to keep it as is, then I would write this as:

  defp handle_frames_body([{:binary, binary}, {:close, _, _}]), do: [binary]
  defp handle_frames_body([{:binary, binary}]), do: [binary]
  defp handle_frames_body([{:close, _, _}]), do: []
  defp handle_frames_body([]), do: []

Anything else looks suspicious :)